### PR TITLE
Sanitize WP only.

### PR DESCRIPTION
**Pull request by fchirica**
Add an option for blueboxes to pick only CC_SP and CC_IP needed for weight proof calculation, as opposed to sanitising all of them. Running only a few in all network should make sure weight proof has all its data blueboxed.

### CODE REVIEW

**Comment by fchirica** [comment]
Passing under test_simulations should give some basic coverage. The next step should be testing on testnet.

File: src/full_node/full_node.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ async def regular_start(self):
                 self.broadcast_uncompact_blocks(
                     self.config["send_uncompact_interval"],
                     self.config["target_uncompact_proofs"],
+                    self.config["sanitize_weight_proof_only"],
```
Comment (mariano54) [inline src/full_node/full_node.py:139]: This is not going to work for people, since we are no longer doing migrations of config. We need to make sure this works if the config variable is not set.

File: src/timelord/timelord.py [hunk 2]
```diff
@@ -72,9 +72,12 @@ def __init__(self, root_path, config: Dict, constants: ConsensusConstants):
         # For each iteration submitted, know if it's a signage point, an infusion point or an end of slot.
         self.iteration_to_proof_type: Dict[uint64, IterationType] = {}
         # List of proofs finished.
-        self.proofs_finished: List[Tuple[Chain, VDFInfo, VDFProof]] = []
+        self.proofs_finished: List[Tuple[Chain, VDFInfo, VDFProof, int]] = []
```
Comment (mariano54) [inline src/timelord/timelord.py:75]: Do you mind making a separate PR with these timelord changes? 
BTW with pycharm you can easily switch between branches with the git GUI. It's very convenient.

File: src/full_node/full_node.py (hunk 1 above)
Comment (fchirica) [reply src/full_node/full_node.py:139]: ok I can default it to False I guess.

**Review by mariano54 — APPROVED** [review]

**Review by hoffmang9 — APPROVED** [review]
+1
